spiffworkflow-backend/bin/clone_process_models

```diff
@@ -24,18 +24,23 @@ fi
 
 if [[ -z "${SPIFFWORKFLOW_BACKEND_GIT_SSH_PRIVATE_KEY_PATH:-}" ]]; then
   if [[ -n "${SPIFFWORKFLOW_BACKEND_GIT_SSH_PRIVATE_KEY:-}" ]]; then
-    export SPIFFWORKFLOW_BACKEND_GIT_SSH_PRIVATE_KEY_PATH=$(mktemp /tmp/ssh_private_key.XXXXXX)
+    SPIFFWORKFLOW_BACKEND_GIT_SSH_PRIVATE_KEY_PATH=$(mktemp /tmp/ssh_private_key.XXXXXX)
```

Comment:
CodeRabbit is missing a subtlety here that shellcheck.net caught: If you export on the same line that you run a subshell, you might mask an error in the execution of the subshell. Not likely here with mktemp, but good BASH style says the export should happen on a separate line from the assignment for that reason.